src/containers/LoginScreen/index.js

```diff
@@ -1,26 +1,66 @@
 import React from 'react';
-import { View, Text } from 'react-native';
+import { View, Text, ImageBackground, TouchableOpacity } from 'react-native';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
 
 import styles from './styles';
+import LoginForm from '../../components/LoginForm';
+import { login } from '../../actions/userActions';
 
 class LoginScreen extends React.Component {
   constructor(props) {
     super(props);
 
-    this.login = this.login.bind(this);
+    this.signIn = this.signIn.bind(this);
   }
 
-  login() {
-    // code for login to be added later
+  signIn = (userData) => {
+    const user = {
+      user: userData.toJS()
+    };
+    return this.props.loginUser(user);
   }
 
   render() {
     return (
-      <View style={styles.color}>
-        <Text>Login</Text>
+      <View style={styles.screen}>
+        <ImageBackground
+          style={styles.backgroundImage}
+          source={require('../../assets/images/group.png')}
+        >
+          <View style={styles.titleView}>
```

Comment:
I think this isn't necessary. Text should be aligning center by itself since `screen` and `backgroundImage` both have `alignItems: 'center'` if it doesn't work try applying `textAlign: 'center'` to the text element.